themes/mlibrary_new/items/show.php

```diff
@@ -1,67 +1,106 @@
 <?php
-fire_plugin_hook('public_items_show', array('view' => $this, 'item' => $item));
-
-// The page isn't injected by the controller
-$page = get_db()->getTable('ExhibitPage')->find($_GET['page']);
+ /**
+  * Copyright (c) 2016, Regents of the University of Michigan.
+  * All rights reserved. See LICENSE.txt for details.
+  */
+  $item_title = strip_formatting(metadata('item', array('Dublin Core', 'Title')));
+  if (empty($item_title)) { $item_title = __('[Untitled]'); }
+  echo head(
+    array(
+      'title'     => $item_title,
+      'bodyid'    => 'items',
+      'bodyclass' => 'show item'
+    )
+  );
+ $exhibit_page = get_record_by_id('exhibit_page',html_escape($_GET['page']));
+  echo '<div class="exhibit-item-back-button"><a href="' .
+        exhibit_builder_exhibit_uri(
```

Comment:
`exhibit_builder_exhibit_uri` should probably be wrapped in `html_escape` before printing it to the page.